PurchasesTests/Mocks/MockHTTPClient.swift

```diff
@@ -3,9 +3,7 @@
 class MockHTTPClient: HTTPClient {
```

Comment:
This has converged a lot with `BackendTests.MockHTTPClient`. I'll be removing the later in a future PR to just use this one.